Assets/HoloToolkit/Utilities/Scripts/WorldAnchorManager.cs

```diff
@@ -141,27 +141,31 @@ public void RemoveAnchor(GameObject gameObjectToUnanchor)
         /// </summary>
         public void RemoveAllAnchors()
         {
+            SpatialMappingManager spatialMappingManager = SpatialMappingManager.Instance;
```

Comment:
Why cache if we're just going to use it a couple of times within the same method?